odoo/addons/base/models/res_bank.py

```diff
@@ -126,3 +126,16 @@ def _condition_to_sql(self, alias: str, fname: str, operator: str, value, query)
             else:
                 value = sanitize_account_number(value)
         return super()._condition_to_sql(alias, fname, operator, value, query)
+
+    def action_archive_bank(self):
+        self.ensure_one()
```

Comment:
May I ask you to add a small docstring to explain why do we have a "custom" archive here.